src/components/TimeBadge.vue

```diff
@@ -16,103 +16,109 @@ export default {
   props: {
     starttime: {
       type: String,
-      default: '1970-01-01T00:00:00.000Z',
```

Comment:
all of the changes in this file were unnecessary, and if there were any, they were lost in the comma removals